### DISCUSSION TITLE
Include a Toast component?

### DISCUSSION THREAD

**Discussion by geoff-harper** [primer/react#2400] [Ideas]
I'm curious where the maintainers are with including a Toast component. It seems that the CSS lib has styles for it and the design docs definitely talk about its usage, but it's omitted here. There also seems to be a PR https://github.com/primer/react/pull/938 that had some headway but has died down. What's the appetite looking like for some Toast 🍞

**Comment by lesliecdubs** [comment]
👋🏻 Hi @geoff-harper, thanks for stopping by! We do have some prior art in Primer CSS for a Toast component, but haven't prioritized work on it for Primer React at the moment. Unfortunately I don't have an ETA to share on it.

If you're willing to share, we'd love to hear more about what you're building with Primer React and what use case you have for a Toast component 👀 

cc @justinbyo for visibility

**Reply by geoff-harper** [reply]
Hey Leslie, thanks for the update. I'm working on a web app with several concurrent webgl environments and some background processes. There's a lot of async stuff going on with unpredictable timings and whathaveyou which is why I was looking to reach for the toast 🍞

**Reply by lesliecdubs** [reply]
Thanks for sharing your use case for 🍞! We'll make note internally of this request. We reevaluate component priority every quarter of the financial year.
